Languages/IronPython/Tests/test_privateBinding.py

```diff
@@ -129,6 +129,6 @@ def test_misc_coverage():
 
 run_test(__name__, noOutputPlease=True)
 
-if not privateBinding and not is_silverlight:
+if not privateBinding and not is_silverlight and not is_netstandard: # sys.executable isn't an executable
```

Comment:
So, if I want to run an ipy console, is it still a DLL? Sorry for the questions, I've never used .NET Core, so I don't know capabilities.